plugins/pom.xml

```diff
@@ -31,5 +31,32 @@
          <artifactId>jboss-logging</artifactId>
          <version>3.3.0.Final</version>
       </dependency>
+
+      <!-- needed to avoid receive NoClassDefFoundError: com/arjuna/ats/arjuna/common/arjPropertyManager
+           during build -->
+      <dependency>
+         <groupId>org.jboss.narayana.jta</groupId>
+         <artifactId>narayana-jta</artifactId>
+         <version>5.10.4.Final</version>
```

Comment:
I think this should either use `${version.jboss.narayana}` or get the property from here: https://github.com/infinispan/infinispan/blob/5e1864da273a448f111629c2029f89c6fe5bf07f/build-configuration/pom.xml#L168

Reply:
Unfortunately we can't use the property from here https://github.com/infinispan/infinispan/blob/5e1864da273a448f111629c2029f89c6fe5bf07f/build-configuration/pom.xml#L168` because we are not using `infinispan-bom` as parent, only as imported.
I'll create `version.jboss.narayana` in the properties